.github/workflows/continuous-integration.yml

```
@@ -51,8 +51,16 @@ jobs:
           dependency-versions: "${{ matrix.dependencies }}"
           composer-options: "${{ inputs.composer-options }}"
 
-      - name: "Run PHPUnit"
+      - name: "Run PHPUnit for PHP 7x"
+        if: "${{ matrix.php-version != '8.0' }}"
```

This seems more future proof:

```suggestion
        if: "${{ startsWith(matrix.php-version, '7') }}"
```

---

indeed!